targets/xliff-tasks.targets

```diff
@@ -3,9 +3,14 @@
   <Target Name="Build">
     <Message Importance="High" Text="Building '$(RepositoryName)'" />
 
-    <Exec Command="$(DotNetCommand) restore $(ProjectDirectory)src\XliffTasks\XliffTasks.csproj" />
+    <!-- Build only the tasks project, not the tests. -->
+    <PropertyGroup>
+      <_XliffTasksProject>$(ProjectDirectory)src\XliffTasks\XliffTasks.csproj</_XliffTasksProject>
+    </PropertyGroup>
 
-    <Exec Command="$(DotNetCommand) build $(ProjectDirectory)build\build.proj /t:Pack /v:normal /flp:Verbosity=Diag;LogFile=$(ProjectDirectory)msbuild.log"
+    <Exec Command="$(DotNetCommand) restore $(_XliffTasksProject)" />
```

Comment:
Isn't `dotnet restore` implicit now?

Reply:
It is. I could collapse this to just pack.

Reply:
Done.